src/Sections/AddPatient/add-patient.tsx

```diff
@@ -1,5 +1,7 @@
 import TextInput from "../../Components/Form/InputText/text-input";
 import Label from "../../Components/Form/Label/label";
+import Contenedor from "../../Components/Form/Contenedor/Contenedor";
```

Comment:
```suggestion
import Contenedor from "../../Components/Form/Contenedor";
```
La idea de tener un index es que puedas llamar a la carpeta desde otras rutas y solo sea visible lo que index exponga (en este caso, un componente del mismo nombre que la carpeta). Esto lo hace por defecto react, importar un index dentro de la carpeta.

El motivo de no llamar directamente al archivo con el componente es que en ocasiones podría exponer ciertas estructuras que quizá sean útiles para pruebas por dar un ejemplo, y que no sean necesarias exponer para usar tal componte, en ese caso el index solo expondrá lo que realmente haga falta